deepdrivemd/config.py

```diff
@@ -220,6 +224,17 @@ def experiment_directory_cannot_exist(cls, v):
         return v
 
 
+class StreamingExperimentConfig(ExperimentConfig):
+    adios_xml_sim: Path
+    adios_xml_agg: Path
+    config_directory: Path
+    software_directory: Path
```

Comment:
It seems like this variable is not used. We should be able to remove it.

Reply:
Software directory is heavily used. Look into yaml file. 

(/usr/workspace/cv_ddmd/conda1/powerai) [yakushin@lassen708:test1_stream]$ grep software_directory *.py  
    software_directory = (
pythonpath = f"{header.software_directory}/misc/:" + os.getenv("PYTHONPATH")
    arguments = [f"{header.software_directory}/sim/openmm_stream/run_openmm.py"]
    arguments = [f"{header.software_directory}/aggregation/stream/aggregator.py"]
    arguments = [f"{header.software_directory}/models/keras_cvae_stream/train.py"]
    arguments = [f"{header.software_directory}/agents/stream/dbscan.py"]

Reply:
It is used to construct YAML files, but not as part of the DeepDriveMD package.